lib/coqProject_file.ml

```diff
@@ -225,12 +225,12 @@ let process_cmd_line ~warning_fn orig_dir parse_extra proj args =
     error "Use \"-arg -impredicative-set\" instead of \"-impredicative-set\""
 
   | "-Q" :: d :: lp :: r ->
-    let lp = if String.equal lp "Coq" then "Stdlib" else lp in
+    let lp = if String.equal lp "Coq" then "Corelib" else lp in
     aux { proj with q_includes = proj.q_includes @ [sourced (mk_path d,lp)] } r
   | "-I" :: d :: r ->
     aux { proj with ml_includes = proj.ml_includes @ [sourced (mk_path d)] } r
   | "-R" :: d :: lp :: r ->
-    let lp = if String.equal lp "Coq" then "Stdlib" else lp in
+    let lp = if String.equal lp "Coq" then "Corelib" else lp in
```

Comment:
I am unsure what this code is about, but I'm wondering if it is correct to replace `Stdlib` with `Corelib`. Isn't this saying that if someone adds `-Q Coq /path/to/stdlib` in their `_CoqProject`, it will get replaced by `-Q Corelib /path/to/stdlib`? It would make more sense to me to keep replacing by `-Q Stdlib /path/to/stdlib`, especially given that Stdlib wrappers around `Corelib` exist. Further, I wonder why this replacement was needed in the first place? What was the actual use case?

Reply:
maybe put a comment explaining that then

Reply:
LGTM; there is minimal hardcoding of the "Coq" prefix in Dune (in anticipation of having stdlib as a regular library), shouldn't be hard to remove it once we install `dune-package` files.

Reply:
Could you please elaborate? (possibly in the Zulip thread if you don't want to lengthen the discussion here)